src/eutils.jl

```diff
@@ -130,7 +130,7 @@ datetype, reldate, mindate, maxdate.
 """
 function elink(ctx::Associative=empty_context(); params...)
     params = process_parameters(params, ctx)
-    return Requests.post(string(baseURL, "elink.fcgi"), data=params)
+    return HTTP.request("POST", string(baseURL, "elink.fcgi"), data=params)
 end
 
 """
```

Comment:
I don't think that HTTP.post accepts a parameter input 'data'. I thought it expects `body` instead. Are we sure the tests are working? They may be responding with 200, but not containing the desired content in the response. Maybe we should improve tests a little bit to check the content more in depth?